main.py

```diff
@@ -330,6 +350,20 @@ def runCommand(command):
     else: 
         log('ERR: RCON has not been correctly initialized!')
 
+def runDebug():
+    print('debug') # DO NOT DELETE THIS LINE!
+    root_dir = os.path.dirname(__file__)
+    server_dir = os.path.join(root_dir, 'server')
+    config = os.path.join(root_dir, 'config.yml')
+
+    if os.path.isfile(config):
+        os.system(f'rm {config}')
+
+    if os.path.isdir(server_dir):
+        os.system(f'rm -rf {server_dir}')
+
+    run()
```

Comment:
Should we not empty this out? We don't want to delete the user's config and server directory when they run the debug option, especially if they don't realize what its meant for.

Reply:
Remember that this is the debug function and is meant to allow you to easily test this change by running `python main.py -D`. Given that this is testing the automatic EULA signing, this was the simplest approach for testing. 

**This should never be run in production**

Reply:
Thats fair. Which is why I wanted to ask. Will we make a plan how to deal with this in production? Should we add a step to empty it or comment it out, or whatever, in our release steps?